gowrapper/main.go

```diff
@@ -205,6 +206,10 @@ type jsDevice struct {
 	readChan chan<- []byte
 }
 
+func (device *jsDevice) Version() string {
```

Comment:
this is new... but how is it related to cardano?

Reply:
Ah, it is needed in Adalite. I split it into a new commit.